src/PanelTraits/Create.php

```diff
@@ -120,4 +144,94 @@ public function syncPivot($model, $data, $form = 'create')
             }
         }
     }
+
+    /**
+     * Create any existing one to one relations for the current model from the form data.
+     *
+     * @param Model $item The current CRUD model.
+     * @param array $data The form data.
+     * @param string $form Optional form type. Can be either 'create', 'update' or 'both'. Default is 'create'.
+     */
+    private function createOneToOneRelations($item, $data, $form = 'create')
+    {
+        $relationData = $this->getRelationDataFromFormData($data, $form);
+        $this->createRelationsForItem($item, $relationData);
+    }
+
+    /**
+     * Create any existing one to one relations for the current model from the relation data.
+     *
+     * @param Model $item The current CRUD model.
+     * @param array $formattedData The form data.
+     */
+    private function createRelationsForItem($item, $formattedData)
+    {
+        foreach ($formattedData['relations'] as $relationMethod => $relationData) {
+            $model = $relationData['model'];
+            $relation = $item->{$relationMethod}();
+
+            if ($relation instanceof BelongsTo) {
+                $modelInstance = $model::find($relationData['values'])->first();
+                if ($modelInstance != null) {
+                    $relation->associate($modelInstance)->save();
+                } else {
+                    $relation->dissociate()->save();
+                }
+            } elseif ($relation instanceof HasOne) {
+                if ($item->{$relationMethod} != null) {
+                    $item->{$relationMethod}->update($relationData['values']);
+                    $modelInstance = $item->{$relationMethod};
+                } else {
+                    $relationModel = new $model();
+                    $modelInstance = $relationModel->create($relationData['values']);
+                    $relation->save($modelInstance);
+                }
+            } else {
+                $relationModel = new $model();
+                $modelInstance = $relationModel->create($relationData['values']);
+                $relation->save($modelInstance);
+            }
+
+            if (isset($relationData['relations'])) {
+                $this->createRelationsForItem($modelInstance, ['relations' => $relationData['relations']]);
+            }
+        }
+    }
+
+    /**
+     * Get a relation data array from the form data. For each relation defined in the fields through the entity
```

Comment:
That seems a bit too long for the first line of a doc block (and I'm being picky). You could probably just put a NL after the first full-stop.